### PR TITLE
Upload perf test results to the CI database

### PR DESCRIPTION
**Pull request by akuzm**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docker/test/performance-comparison/compare.sh [hunk 1]
```diff
@@ -1074,6 +1074,53 @@ wait
...
 
+function upload_results
+{
+    if ! [ -v CHPC_DATABASE_URL ]
+    then
+        echo Database for test results is not specified, will not upload them.
+        return 0
+    fi 
+
+    # Surprisingly, clickhouse-client doesn't understand --host 127.0.0.1:9000
```
Comment (alexey-milovidov) [inline docker/test/performance-comparison/compare.sh:1085]: You can use `SocketAddress` for this purpose.
Just don't forget to test all the cases:

`127.0.0.1:9000`
`[2a02:6b8:a::a]:9000`
`[::]:9000`
`[::ffff:127.0.0.1]:9000`

Comment (alexey-milovidov) [reply docker/test/performance-comparison/compare.sh:1085]: And also to propagate it to all clickhouse tools.

**Review by alexey-milovidov — APPROVED** [review]
Ok.

I'd also upload raw query run times from all runs (so we can compare with various methods).
Also it will be nice to upload all trace_log and query_log (but we need to upgrade our server in Yandex.Cloud to something reasonably powerful). And metric_log and asynchronous_metric_log if you feel greedy.

**Comment by akuzm** [comment]
> Ok.
> 
> I'd also upload raw query run times from all runs (so we can compare with various methods).
> Also it will be nice to upload all trace_log and query_log (but we need to upgrade our server in Yandex.Cloud to something reasonably powerful). And metric_log and asynchronous_metric_log if you feel greedy.

Query metrics for each run are about 300 MB of uncompressed TSV, so about 30 MB compressed, 600 MB daily (for master) and 20 GB monthly. All logs and metrics are also about 400 MB of uncompressed TSV together.

I'll add them in the next PR.

**Comment by akuzm** [comment]
It still doesn't work -- now the robot doesn't have enough priviliges -- but I think it's good enough to merge. The test failures are not related, because I'm only touching the perf test.

**Pull request merged**